_posts/18-12-03-vido: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  "2018-12-02-Reading: 7 reasons why smart, hardworking people don't become successful"
+date:   2018-12-02 19:31:07 +0800
+categories: readingnote
+---
+
+[Video Link](2018-12-02-video: https://www.youtube.com/watch?v=cqINcpbcZqs)
+
+#### Summary:
+The seven reasons the auther have offered us is:
+
+1、You don't reach out to new people.
+2、You are averse to change.
+3、You're not willing to take risks.
+4、You believe you deserve success based on credentials.
+5、You constantly go after whatever's exciting at the moment.
+6、You can't commit to a decision.
+7、You don't believe in yourself.
+
+#### Reflection:
+The video is really refreshing and there are really some areas I need to improve.
+
+Firstly, make new friends especially from different areas, have different life experience or have different ideas. Start by say hi to strangers, welcoming different ideas, opinions or even unique perspective etc.
+
+Secondly, go to different public event, try new things or go experience different life-style.
```

Comment:
event => events
try new things or go to... => try new things```,``` or go to... ( proper listing grammar is "a, b, or c", or "a, b, and c". You should always put common before or when listing more than 2 items.)